src/translations_reader.py

```diff
@@ -0,0 +1,38 @@
+from src.word import Word
+from conllu import parse_incr
+from src.sentence import Sentence
+from src.translation import Translation
+
+
+def read_conllu_outputs(reference_path, hypothesis_path):
+    translations = []
+    with open(reference_path, 'r', encoding="utf-8") as ref, open(hypothesis_path, 'r', encoding="utf-8") as hyp:
+        for ref_tokenlist, hyp_tokenlist in zip(parse_incr(ref), parse_incr(hyp)):
```

Comment:
the convention is not to write the type of the variable in its name
ref_tokenlist -> ref_tokens